cpp/basix/lattice.cpp

```diff
@@ -156,14 +156,13 @@ xt::xtensor<double, 2> tabulate_dlagrange(int n,
   xt::xtensor<double, 3> dual_values
       = polyset::tabulate(cell::type::interval, n, 0, equi_pts);
   xt::xtensor<double, 2> dualmat({dual_values.shape(2), dual_values.shape(1)});
-  dualmat.assign(xt::transpose(xt::view(dual_values, 0, xt::all(), xt::all())));
+  dualmat.assign(xt::view(dual_values, 0, xt::all(), xt::all()));
 
   xt::xtensor<double, 3> tabulated_values
       = polyset::tabulate(cell::type::interval, n, 0, x);
   xt::xtensor<double, 2> tabulated(
       {tabulated_values.shape(2), tabulated_values.shape(1)});
```

Comment:
Is this correct when everything else has been transposed?

Reply:
I think it's ok